src/envoy/http/authn/http_filter.cc

```diff
@@ -78,6 +85,44 @@ FilterHeadersStatus AuthenticationFilter::decodeHeaders(HeaderMap&, bool) {
     }
   }
 
+  int origins_size = 0;
+  // In POC, only inspect the first credential_rule
+  if (config_.credential_rules_size() > 0 &&
+      config_.credential_rules()[0].origins_size() > 0) {
+    origins_size = config_.credential_rules()[0].origins_size();
+  }
+  ENVOY_LOG(debug, "AuthenticationFilter: {} config.origins_size()={}",
+            __func__, origins_size);
+  if (origins_size > 0) {
```

Comment:
this is not need:  jwt_store_factory is null

Reply:
Done.